src/lib/components/dropdown-button/stories/index.stories.tsx

```diff
@@ -598,23 +536,23 @@ export const WithoutLabelOnlyIcon: Story = {
 
       expect(
         firstChildLength + middleChildLength + lastChildLength,
-        'It expects that the dropdown has 9 elements'
-      ).toBe(9);
+        'It expects that the dropdown has 6 elements'
+      ).toBe(6);
 
       expect(
         dropdown,
         `It expects that the dropdown first child has the title Option 1`
-      ).toHaveTextContent('List item');
+      ).toHaveTextContent('Option 1');
 
       expect(
-        textElements,
-        'It expects that the dropdown has 3 text elements'
-      ).toHaveLength(3);
+        dropdown,
+        `It expects that the dropdown first child has the title Option 2`
+      ).toHaveTextContent('Option 2');
```

Comment:
there seems to be no difference in the way you take the elements from your dropdown, and this doesn't validate the test